crates/bevy_core_pipeline/src/smaa/smaa.wgsl

```diff
@@ -444,7 +444,7 @@ const SMAA_LOCAL_CONTRAST_ADAPTATION_FACTOR: f32 = 2.0;
 const SMAA_AREATEX_MAX_DISTANCE: f32 = 16.0;
 const SMAA_AREATEX_MAX_DISTANCE_DIAG: f32 = 20.0;
 const SMAA_AREATEX_PIXEL_SIZE: vec2<f32> = (1.0 / vec2<f32>(160.0, 560.0));
-const SMAA_AREATEX_SUBTEX_SIZE: f32 = (1.0 / 7.0);
+const SMAA_AREATEX_SUBTEXTURE_SIZE: f32 = (1.0 / 7.0);
```

Comment:
this might be texel not texture. id need to read the associated papers or implementations, id leave this alone if i were you. it says TEX in a few other places and id rather stick to whatever convention the papers follow